Fix small typo for ceil behavior docs

codebycaleb opened the pull request:
I got caught off-guard by the existing documentation, so I just wanted to submit a quick fix for it. Would be happy to switch to ```Returns the smallest integer greater than or equal to `number`.``` if that sounds better!

josevalim commented:
Wouldn’t greater be better to avoid the negation? How are the docs for
floor? I would just mirror that.
-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

codebycaleb commented:
Good question! Floor docs also contain the negation:

`Returns the largest integer not greater than number`

I've removed both negations and squashed my commits together. 🙂

josevalim commented:
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
